hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SnapshotChainManager.java

```diff
@@ -71,7 +71,27 @@ public SnapshotChainManager(OMMetadataManager metadataManager) {
   private void addSnapshotGlobal(UUID snapshotID,
                                  UUID prevGlobalID) throws IOException {
     // On add snapshot, set previous snapshot entry nextSnapshotID = snapshotID
+    if (globalSnapshotChain.containsKey(snapshotID)) {
+      throw new IllegalStateException(String.format(
+          "Snapshot chain corruption. Snapshot with snapshotId: %s is " +
```

Comment:
nit: Can you please change exception message to `Global snapshot chain corruption` for all the exception messages in `addSnapshotGlobal` and `Path level snapshot chain corruption` for the exception messages in `addSnapshotPath`? And assert that exception message starts with correct message in the test.

Reply:
done